R/three-legged-oauth.R

```diff
@@ -1,8 +1,23 @@
-
-#' Create a token for a user via the browser flow.
+#' Get an OAuth token for a user
+#'
+#' @description Consults the token cache for a suitable OAuth token and, if
+#'   unsuccessful, gets a token via the browser flow. A cached token is suitable
+#'   if it's compatible with the user's request in this sense:
+#'   * OAuth app must be same.
+#'   * Scopes. This is a check for inclusion, not exact equality.
+#'   * Email, if provided, must be same.
+#'
+#' @description If multiple suitable tokens are found, user is presented with a
```

Comment:
You only need one `@description`

Reply:
But I want to keep this text in the Description vs Details:

https://github.com/jeroen/commonmark/issues/7

Reply:
A roxygen tag affects all text until the next tag - so the first `@description` should be adequate.

Reply:
I stand corrected! I wonder what has changed? Because you're right -- I don't need this now but I did at the time of that issue 🤔 Removing the second `@description` tag here.